Laravel-Backpack/CRUD
# Remove backpack error pages from user land

## WHY

### BEFORE - What was wrong? What was happening before this PR?

We were "hijacking" the laravel error pages to display them with Backpack layouts. 
For the intended behavior (displaying nice error messages in backpack admin panel) it worked great, but it also meant that those error views were displaying when accessing urls outside of the admin panel, and that's a side effect, not the desired behavior.

### AFTER - What is happening after this PR?

We no longer use the Laravel error pages as part of the admin panel.
Backpack now provides it's own error pages as part of the ui.
Those error pages will only be displayed in case **user is logged in** and the **url is inside `/admin/`** (route prefix name), otherwise the default Laravel error pages will be served. 

Those error pages can be overwritten on a theme basis. 

## HOW

### How did you achieve that, in technical terms?

Removed errors from default Laravel location and added a `ui/errors` folder for it.
Added theme paths and ui paths to error reporting. 


### Is it a breaking change?

yes, unfortunately I think it is and should be added to the upgrade guide. 
mainly the folder structure and the usage of `backpack_view()` helpers to have the fallbacks available. 

### How can we test the before & after?

There is nothing "theoretically" broken before.

But to test you can check that if any error is thrown in an "admin page", the error will use the theme layout to display it nicely to the user (when themed, at the moment only the "default ui errors" are provided in this PR.

You can add `abort(500,'test')` in any crud to test it. 
You can throw the same error outside of `admin/` routes, for example editing your `routes/web.php` and adding:

```php
Route::get('/', function () {
    abort(500, 'test');
});
```

You will see that now the Laravel error is shown to the user, as it should, while previously it would should the "backpack error page".
